config/deployer.sample.json

```diff
@@ -33,5 +33,30 @@
     "FileLevel": "INFO",
     "FileJson": true,
     "FileLocation": "deployer.log"
+  },
+  "Report": {
+    "Label": "{instance=~\"app.*\"}",
+    "GraphQueries": [
+      {
+        "Name":  "CPU Utilization",
+        "Query": "avg(irate(mattermost_process_cpu_seconds_total{instance=~\"app.*\"}[1m])* 100)"
+      },
+      {
+        "Name":  "Heap In Use",
+        "Query": "avg(go_memstats_heap_inuse_bytes{instance=~\"app.*:8067\"})"
+      },
+      {
+        "Name":  "Stack In Use",
+        "Query": "avg(go_memstats_stack_inuse_bytes{instance=~\"app.*:8067\"})"
+      },
+      {
+        "Name":  "RPS",
+        "Query": "sum(rate(mattermost_http_requests_total{instance=~\"app.*:8067\"}[1m]))"
+      },
+      {
+        "Name":  "Avg Store times",
+        "Query": "sum(increase(mattermost_db_store_time_sum{instance=~\"app.*:8067\"}[1m])) / sum(increase(mattermost_db_store_time_count{instance=~\"app.*:8067\"}[1m]))"
```

Comment:
Here and in a few other places I'd switch to use `rate` instead of `increase` since we are dividing to calculate an average.

Reply:
I thought this is the correct way to calculate average? We are dividing total time taken by the count of events. And this is the way it's implemented in Grafana right now.

You want `sum(rate()) / sum(rate())`?

Reply:
Ah you want rate()/rate() right?

Reply:
Correct, the result is the same really but it doesn't make sense to use increase which means an additional multiplication for the number of seconds in the time range.